src/java.base/share/classes/java/net/Socket.java

```diff
@@ -1733,8 +1733,9 @@ public boolean getReuseAddress() throws SocketException {
      * will throw a {@link SocketException}.
      * <p>
      * Once a socket has been closed, it is not available for further networking
-     * use (i.e. can't be reconnected or rebound). A new socket needs to be
-     * created.
+     * use (i.e. can't be reconnected or rebound) and several of the methods defined
+     * by this class will throw an exception when invoked on the closed socket. A new
```

Comment:
I think "if invoked" might be better than "when invoked" here

Reply:
Fixed and updated the PR to address both these places.